apps/db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/InntektsmeldingRepository.kt

```diff
@@ -130,17 +130,16 @@ class InntektsmeldingRepository(
         inntektsmeldingId: UUID,
         inntektsmeldingSkjema: SkjemaInntektsmelding,
         mottatt: LocalDateTime,
-    ): Long =
-        Metrics.dbInntektsmelding.recordTime(InntektsmeldingRepository::lagreInntektsmeldingSkjema) {
```

Comment:
Antar at denne metrikken ikke fungerer lengre?

Reply:
Nei, den forsvant for lenge siden. Gjelder egentlig en haug med metrikker.